spoe_auth_snap/hooks/bin/default-configure

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Missing copyright header. Not sure why it isn't pointed out by the workflow

Reply:
updated